### ISSUE TITLE
Optimize the password reset flow in the admin panel

### ISSUE THREAD

**Issue by alachaum** [KeypDemoCorp/cooking-app-frontend#20]
We should align this with the rest of the application for a consistent experience.

Accessibility should be considered when implementing this.
